reactor-core/src/test/java/reactor/core/publisher/FluxReplayTest.java

```diff
@@ -59,6 +62,80 @@ public void vtsStop() {
 		VirtualTimeScheduler.reset();
 	}
 
+	static final Logger log = Loggers.getLogger(FluxReplayTest.class);
+
+	@Test
+	void test() {
```

Comment:
please give this test a more demonstrative name

Reply:
FIXED